src/app/collective-rewards/rewards/MyRewards.tsx

```diff
@@ -26,20 +32,91 @@ const SubText = () => {
   )
 }
 
-export const Rewards: FC<{ builder: Address }> = ({ builder }) => {
+const RewardsContent: FC<RewardDetails> = data => {
+  const { builder } = data
+
+  // We don't need to show the loading state for the backer rewards since the parent already has a loading state
+  const { data: isBacker, error: backerError } = useIsBacker(builder)
+  const { data: gauge, error: gaugeError } = useGetBuilderToGauge(builder)
+
+  const error = gaugeError ?? backerError
+
+  useHandleErrors({ error, title: 'Error loading gauge / isBacker' })
```

Comment:
I'd change the title to something simpler, e.g.: "Error loading user data" or something similar.